datadog.gemspec

```diff
@@ -65,7 +65,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'datadog-ruby_core_source', '~> 3.3'
 
   # Used by appsec
-  spec.add_dependency 'libddwaf', '~> 1.15.0.0.0'
+  spec.add_dependency 'libddwaf', '~> 1.18.0.0.0'
```

Comment:
I believe we're planning a release for later today, so reminder that if you press that merge button, this is going to go out today. (You're probably aware, but just in case!)